crowbar_framework/app/models/node_object.rb

```diff
@@ -1451,6 +1451,26 @@ def disk_release!(device, owner)
     disk_release(device, owner) and save
   end
 
+  def verify_disks_claim
+    unclaimed_disks = Array.new
```

Comment:
This should also be renamed. This disks returned are not unclaimed. They are claimed but do not refer to any existing device on the node.